packages/legacy/src/Providers/components/AddEditProviderModal/AddEditProviderModal.tsx

```diff
@@ -435,6 +486,57 @@ export const AddEditProviderModal: React.FunctionComponent<IAddEditProviderModal
                     />
                   </FormGroup>
                 ) : null}
+                {fields?.insecure? (
+                  <Checkbox
+                    label="Insecure connection"
```

Comment:
```suggestion
                    label="Insecure connection, If checked, the server's certificate will not be checked for validity."
```

Reply:
After a discussion with Arik, I rephrased it to:

![image](https://user-images.githubusercontent.com/18169498/219389942-a2016cc6-c3ad-4b99-9146-49d33177b545.png)

The actual action is that if a certificate is not given, a local one is verified. So it's not really an insecure connection like in oVirt. It's more an option to skip verifying a given certificate but the connection might still be secured.